registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -244,6 +245,7 @@
     "sha256:0451a69e9ed436840a0538169f7126e5d1175906a74bf273f3d035515f4b87d8": ["v1.10.2"]
     "sha256:2294ae6670c0d0efb19949f4a05c169cb90b995145a9e6e7c04bc068583fff6e": ["v1.10.3"]
     "sha256:3a8e859273778370962d8ad9e496d071b414202bc13aef8e06e2459c836b7449": ["v1.10.4"]
+    "sha256:f588fbe6f73160981cb7ae8ed9d1e48ea0322ddf3ec28125532cb0af01d91ce5": ["v1.10.5"]
```

Comment:
SHA `f588fbe6f731` matches clusterctl v1.10.5 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:f588fbe6f73160981cb7ae8ed9d1e48ea0322ddf3ec28125532cb0af01d91ce5/details?inv=1&invt=Ab56Hw).